src/Console/ModelsCommand.php

```diff
@@ -840,6 +841,20 @@ protected function getSoftDeleteMethods($model)
         }
     }
 
+    /**
+     * Generates methods that return collections
+     * @param \Illuminate\Database\Eloquent\Model $model
+     */
+    protected function getCollectionMethods($model)
+    {
+        $collectionClass = $this->getCollectionClass(get_class($model));
+
+        if ($collectionClass !== '\Illuminate\Database\Eloquent\Collection') {
```

Comment:
```suggestion
        if ($collectionClass !== \Illuminate\Database\Eloquent\Collection::class) {
```

Reply:
@mfn Had to adjust this slightly as when using class reference the leading `\` does not remain.

Reply:
meh, sorry for the troubles; this truly annoying at times; thanks for fixing it 😄